src/java.base/share/classes/java/security/Key.java

```diff
@@ -86,6 +86,10 @@
  * Security Appendix</a>
  * of the <cite>Java Object Serialization Specification</cite> for more information.
  *
+ * @spec serialization/index.html Java Object Serialization Specification
+ * @spec https://www.rfc-editor.org/info/rfc5280
+ *        RFC 5280: Internet X.509 Public Key Infrastructure Certificate
```

Comment:
any reason why you indented this comment with 8 spaces? All others are indented with 6

Reply:
I will change it to be consistent.